src/test/groovy/pl/tomaszdziurko/jvm_bloggers/mailing/BlogPostForMailItemSpec.groovy

```diff
@@ -26,13 +32,31 @@ class BlogPostForMailItemSpec extends Specification {
             Blog author = stubAuthorWith(name, twitter)
             BlogPost post = Stub(BlogPost) {
                 getBlog() >> author
+                getUrl() >> url
+                getTitle() >> title
             }
         when:
             BlogPostForMailItem blogPostForMailItem = BlogPostForMailItem.builder().from(post).build()
         then:
             blogPostForMailItem.authorLabel == "<a href=\"https://twitter.com/" + twitter.substring(1) + "\">" + name + "</a>"
     }
 
+    def "Should store author name as a label if twitter account is not set correctly."() {
```

Comment:
The name of the test is kinda enigmatic. There is no useful information what is really tested here. Consider replacing  _is not set correctly_ into sth. more descriptive.